CI.hs

```diff
@@ -104,10 +115,28 @@ main = do
         cmd "stack sdist ghc --tar-dir=."
         cmd $ "tar -xvf " ++ target ++ ".tar.gz"
 
-      rmFiles :: [FilePath] -> IO ()
-      rmFiles fs = forM_ fs $
-        \f -> removeFile f `catchIOError` (const  $ return ())
+rmFiles :: [FilePath] -> IO ()
+rmFiles fs = forM_ fs $
+  \f -> removeFile f `catchIOError` (const  $ return ())
+
+rmDirs :: [FilePath] -> IO ()
+rmDirs ds = forM_ ds $
+ \d -> removeDirectoryRecursive d `catchIOError` (const $ return ())
+
+tag :: IO String
+tag = do
+  DateTime y m d _ _ _ <- getCurrentDateTime
```

Comment:
Unless there is a specific reason for why we need to use the `dates` package I would recommend to use the much more popular `time` package which is even shipped with GHC.

Reply:
✔️